### PR TITLE
Updated governance and code of conduct documents.

### PR DESCRIPTION
**Pull request by ngokevin**
**Description:**

As commonly requested, this PR outlines the governance with added moderators: 

Project Maintainers: @dmarcos @donmccurdy @ngokevin
Moderators: @AdaRoseEdwards @Utopiah 

For clarity/efficiency and to enforce the explicit governance, we propose to have only maintainers keep admin/write permissions to the @aframevr organization and its repositories (unless provided otherwise such as with @fernandojsg and the Inspector).

The Code of Conduct has been updated to adapt to A-Frame's community, previously copied from Rust. The Code of Conduct has been tweaked and streamlined for A-Frame with some less redundancy.

### CODE REVIEW

**Comment by dmarcos** [comment]
It looks good to me

**Comment by Utopiah** [comment]
Ideally using fabien-services@benetou.fr instead.

File: GOVERNANCE_CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+Governance outlines the set of people that lead the project, its vision, and
+its community as well as detailing how final decisions are carried out. A-Frame
+is governed by a core team of project maintainers, which is ultimately
+responsible for all decision-making in the project. Specifically, project
+maintainers:
+
+- Sets the overall direction and vision for the project
+- Sets the priorities and release schedule
+- Makes final decisions when consensus cannot be reached
+- Upholds the code of conduct
```
Comment (donmccurdy) [inline GOVERNANCE_CODE_OF_CONDUCT.md]: Nit: Should use "set" and "make" to match plural subject.

File: GOVERNANCE_CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+[@dmarcos]: https://github.com/dmarcos
+[@donmccurdy]: https://github.com/donmccurdy
+[@ngokevin]: https://github.com/ngokevin
+
+Project maintainers currently consist:
```
Comment (donmccurdy) [inline GOVERNANCE_CODE_OF_CONDUCT.md]: "Current project maintainers:"

File: GOVERNANCE_CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+- Sustained, significant, and quality contribution.
+- Technical knowledge of 3D, VR, three.js, and A-Frame.
+- Responsive support and engagement to the community.
+- Demonstration of the code of conduct.
+
+[@lady_ada_king]: https://twitter.com/lady_ada_king
+[@utopiah]: https://twitter.com/utopiah
+
+Moderators consist of:
```
Comment (donmccurdy) [inline GOVERNANCE_CODE_OF_CONDUCT.md]: "Current moderators:"

File: GOVERNANCE_CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+- Avoid defaming other people. If you do not have anything nice to say, try to
+  not say it at all.
+- Likewise any spamming, trolling, flaming, baiting or other attention-stealing
+  behaviour is not welcome.
+
+Try not to take any disagreements, code reviews, technical decisions, nor forms
+of discourse personally or as attacks. All decisions made should be made
+without bias towards any of the participants. If you feel this is not being
+upheld, please contact any of the project maintainers or raise your concerns in
+the forums.
```
Comment (donmccurdy) [inline GOVERNANCE_CODE_OF_CONDUCT.md]: any of the project maintainers or moderators

**Comment by fernandojsg** [comment]
lgtm!

**Comment by larsbergstrom** [comment]
Thanks for kicking off the formal discussion of governance, Kevin! As we discussed in the past, this is an issue that has been informal since the project’s inception and I’m excited to see something codified as the project grows and matures.

At a high level, I think A-Frame’s governance ideally ought to take into account existing members and organizations, as well as Mozilla's historical role as the steward. This would also allow Mozilla to continue to invest in protecting and growing the A-Frame brand, services and web properties, and development. 

I’d like to propose having a conversation about governance goals and guidelines in a new GH issue, as ultimately the governance for A-Frame ought to be more comprehensive rather than a few paragraphs in the top-level repo.

I will open a new issue shortly to kick this off.

**Comment by ngokevin** [comment]
Status update for y'all! While the proposal has been accepted by community and members, still waiting on a response from Mozilla executive/legal people that decided to open a separate issue. Hope to hear back from them soon.

**Comment by ngokevin** [comment]
Unfortunately, there hasn't been much updates on this, I am going to close this for now and move to the issue.

**Pull request closed without merging**